django/thunderstore/frontend/context.py

```diff
@@ -0,0 +1,9 @@
+from thunderstore.frontend.models import NavLink
+
+
+def nav_links(request):
+    return {
+        # This is implicitly ordered based on the model's default ordering,
+        # which follows the db index.
```

Comment:
Could this be clarified a bit? I read it wrong the first time, thought it meant that the items are returned based by their DB ids. After reading the model definition and rereading the comment I understand what it says, and it's technically correct, the best kind of correct, but maybe still not ideal?